anchore_engine/services/apiext/api/helpers/image_content_response.py

```diff
@@ -0,0 +1,220 @@
+            el["package"] = content_data[package]["name"]
+            el["type"] = content_data[package]["type"].upper()
+            el["location"] = content_data[package]["location"]
+            el["specification-version"] = content_data[package]["specification-version"]
+            el["implementation-version"] = content_data[package]["implementation-version"]
+            el["maven-version"] = content_data[package]["maven-version"]
+            el["origin"] = content_data[package]["origin"] or "Unknown"
+        except:
+            continue
+        response.append(el)
+    return response
+
+
+def _build_files_response(content_data):
+    response = []
+    elmap = {
+        "linkdst": "linkdest",
+        "size": "size",
+        "mode": "mode",
+        "sha256": "sha256",
+        "type": "type",
+        "uid": "uid",
+        "gid": "gid",
+    }
+    for filename in list(content_data.keys()):
+        el = {}
+        try:
+            el["filename"] = filename
+            for elkey in list(elmap.keys()):
+                try:
+                    el[elmap[elkey]] = content_data[filename][elkey]
+                except:
+                    el[elmap[elkey]] = None
+
+            # special formatting
+            el["mode"] = format(stat.S_IMODE(el["mode"]), "05o")
+            if el["sha256"] == "DIRECTORY_OR_OTHER":
+                el["sha256"] = None
+        except:
+            continue
+        response.append(el)
+    return response
+
+
+def _build_docker_history_response(content_data):
+    try:
+        return utils.ensure_str(
+            base64.encodebytes(utils.ensure_bytes(json.dumps(content_data)))
+        )
+    except Exception as err:
+        logger.warn("could not convert content to json/base64 encode - exception: %s", err)
+    return ""
+
+
+def _build_dockerfile_response(content_data):
+    try:
+        return utils.ensure_str(base64.encodebytes(utils.ensure_bytes(content_data)))
+    except Exception as err:
+        logger.warn("could not base64 encode content - exception: %s", err)
+    return ""
+
+
+def _build_manifest_response(content_data):
+    try:
+        return utils.ensure_str(base64.encodebytes(utils.ensure_bytes(content_data)))
+    except Exception as err:
+        logger.warn("could not base64 encode content - exception: %s", err)
+    return ""
+
+
+def _build_default_response(content_data):
+    response = []
+    try:
+        for package in list(content_data.keys()):
+            el = {}
+            try:
+                el["package"] = content_data[package]["name"]
+                el["type"] = content_data[package]["type"].upper()
+                el["location"] = content_data[package].get("location", None) or "Unknown"
+                el["version"] = content_data[package].get("version", None) or "Unknown"
+                el["origin"] = content_data[package].get("origin", None) or "Unknown"
+                el["license"] = content_data[package].get("license", None) or "Unknown"
```

Comment:
If the line below splits on whitespace, wouldn't this cause multiple licenses to be pilled up together?

Reply:
`el['license']` is intended to have multiple licenses in a single string field (delimited by spaces --presumably). That is the current behavior/expectation for this field today.

`el['licenses']` is intended to be foremost a list type, even if the string parse fails. Assuming a licenses string (e.g. `"MIT GPLv2"`) splitting by space would be sufficient (e.g. `["MIT", "GPLv2"]`). However, if a single license has a space in it, or they are comma delimited, then this default behavior will fail. Note that this is the "default" handler, that is, there is no guide what is "right" here... it's best guess.